src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx

```diff
@@ -187,6 +187,25 @@ const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
 cfnUserPool.addPropertyOverride('Policies.PasswordPolicy.MinimumLength', 32);
 ```
 
+## Customize Cognito UserPool email verification settings
+
+You can change the verification email by overriding the `verificationEmailStyle` and the `verificationEmailBody`.
+
+```ts title="amplify/auth/resource.ts"
```

Comment:
```suggestion
```ts title="amplify/auth/resource.ts"
// amplify/auth/resource.ts
```

Reply:
I think this comment is just backticks, I added it back in.

Reply:
this one is to add a comment at the top of the snippet with a path to the file

Reply:
Ok will do

Reply:
#[c99ffe9](https://github.com/aws-amplify/docs/pull/6516/commits/c99ffe9a7b05513136340406bad22d05b888d5bf)